js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
The second case is a "named blob" which was something Jasper invented: https://github.com/endlessm/eos-knowledge-lib/commit/12e0b3900a37eff58acb8e55cd48a298c69107ba So, I think your check for the component starting with `#` should work fine.

Personally I like your WIP branch better, I prefer not having the overhead of the jquery hack firing on every scroll event, if we don't need it for updating the TOC. But maybe we can ask Joana to weigh in?